### PR TITLE
Remove some sunodo references

### PR DESCRIPTION
**Pull request by tuler**
This removes some references to `sunodo`.
It centralizes for example references to the `.sunodo` directory.

This will facilitate #462

### CODE REVIEW

**Review by omidasadpour — COMMENTED** [review]
I think it would be better to not use type `any` in https://github.com/sunodo/sunodo/pull/487/files#diff-0eda9573885c4c62dc779278a8b0e8dd493db82e70837b1886e71e09062b8a80R150


This one may make sense instead: 

```code
            if (
                e instanceof Error &&
                (e as Error & { exitCode?: number }).exitCode !== 130
            ) {

```

File: apps/cli/src/commands/build.ts [hunk 1]
```diff
@@ -299,23 +299,23 @@ Update your application Dockerfile using one of the templates at https://github.
...
 
             // create machine snapshot
             if (!flags["skip-snapshot"]) {
                 await this.sdkRun(
                     sdkImage,
                     BuildApplication.createMachineSnapshotCommand(imageInfo),
-                    SUNODO_DEFAULT_EXT2_PATH,
-                    SUNODO_DEFAULT_MACHINE_SNAPSHOT_PATH,
+                    ext2Path,
+                    this.getContextPath("image"),
```
Comment (endersonmaia) [inline apps/cli/src/commands/build.ts:312]: For consistency, adding a `const snapshotPath = this.getContextPath("image");` at line 268 and then:

```suggestion
                    snapshotPath,
```

File: apps/cli/src/baseCommand.ts [hunk 2]
```diff
@@ -57,9 +57,13 @@ export abstract class BaseCommand<T extends typeof Command> extends Command {
         return ps?.State;
     }
 
+    protected getContextPath(...paths: string[]): string {
+        return path.join(".sunodo", ...paths);
```
Comment (endersonmaia) [inline apps/cli/src/baseCommand.ts:61]: Could we get the name dynamically like you did wth `${this.config.bin}`, don't know if it is worth but could remove `.sunodo` reference here also.

**Review by endersonmaia — CHANGES_REQUESTED** [review]

File: apps/cli/src/baseCommand.ts (hunk 2 above)
Comment (tuler) [reply apps/cli/src/baseCommand.ts:61]: I think it's enough to just have this single point with `.sunodo`.
Maybe we will have to support both `.sunodo` and `.cartesi` in the transition. Subject for another PR anyway.

**Comment by tuler** [comment]
>             if (
>                 e instanceof Error &&
>                 (e as Error & { exitCode?: number }).exitCode !== 130
>             ) {
> ```

But it's not that better. You are injecting an optional `exitCode` in an object you don't know anyway.

**Pull request merged**